### PR TITLE
Add documentation for new Rainforest Eagle-200 platform

### PR DESCRIPTION
**Pull request by gtdiehl**
**Description:**
New documentation for Rainforest Eagle-200 platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24919
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9761"></a>

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9761"></a>

### CODE REVIEW

File: source/_components/rainforest_eagle.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+logo: rainforest_automation_logo.png
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Energy
+ha_release: 0.96
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/rainforest_eagle.markdown]: Please remove the `redirect_from`

File: source/_components/rainforest_eagle.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+sensor:
+  - platform: rainforest_eagle
+    ip_address: IP_FOR_EAGLE
+    cloud_id: CLOUD_ID_FROM_EAGLE
+    install_code: INSTALL_CODE_FROM_EAGLE
+
+```
+
+```yaml
+# example configuration.yaml entry, limiting the metrics to demand only
```
Comment (klaasnicolaas) [inline source/_components/rainforest_eagle.markdown]: Move this example underneed the configuration variables and give it a full example title

File: source/_components/rainforest_eagle.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+layout: page
+title: "Rainforest Eagle-200"
+description: "Instructions on how to setup the Rainforest Eagle-200 with Home Assistant."
+date: 2019-07-02 18:00
+logo: rainforest_automation_logo.png
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/rainforest_eagle.markdown:5]: Maybe add `sensor`?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/rainforest_eagle.markdown (hunk 1 above)
Comment (gtdiehl) [reply source/_components/rainforest_eagle.markdown]: Removed `redirect_from`

File: source/_components/rainforest_eagle.markdown (hunk 2 above)
Comment (gtdiehl) [reply source/_components/rainforest_eagle.markdown]: Moved the example underneath the configuration variables and created an examples section with two examples.

File: source/_components/rainforest_eagle.markdown (hunk 3 above)
Comment (gtdiehl) [reply source/_components/rainforest_eagle.markdown:5]: Added `Sensor` to `ha_category`

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Comment by gtdiehl** [comment]
I updated the documentation and removed the Examples and variable selectors as I had to remove direct access to them in the code per Architecture issue #100 [https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md](url)

**Review by frenck — APPROVED** [review]
Thanks, @gtdiehl! 👍

**Pull request merged**